.github/workflows/find-helm-chart-by-ticket-or-main-in-registry.yml

```diff
@@ -40,6 +40,7 @@ jobs:
         uses: cardinalby/git-get-release-action@1.2.4
         with:
           releaseNameRegEx: "${{ inputs.chart_name }}-0.0.0-${{ inputs.branch }}"
+          searchLimit: 100000
```

Comment:
I think we could use something lower (like 1000) here, because on branches the chart should be released recently.